catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -691,4 +709,29 @@ class ValidationParserSpecTest extends Specification {
 }
 '''
 
+    String matchOneAttributeValidator = '''
+{
+    "validators": {
```

Comment:
`validators` is a JSON object that has an attribute name (`title`) and an array of validators associated with the attribute name